lib/backends/secrets-manager-backend.js

```diff
@@ -21,11 +21,12 @@ class SecretsManagerBackend extends KVBackend {
    * @param {string} key - Key used to store secret property value in Secrets Manager.
    * @param {object} keyOptions - Options for this specific key, eg version etc.
    * @param {string} keyOptions.versionStage - Version stage
+   * @param {string} keyOptions.versionId - Version ID
    * @param {object} specOptions - Options for this external secret, eg role
    * @param {string} specOptions.roleArn - IAM role arn to assume
    * @returns {Promise} Promise object representing secret property value.
    */
-  async _get ({ key, specOptions: { roleArn }, keyOptions: { versionStage = 'AWSCURRENT' } }) {
+  async _get ({ key, specOptions: { roleArn }, keyOptions: { versionStage = 'AWSCURRENT', versionId } }) {
```

Comment:
Just to make it a bit clearer that its optional

```suggestion
  async _get ({ key, specOptions: { roleArn }, keyOptions: { versionStage = 'AWSCURRENT', versionId = null } }) {
```